backend/common/src/main/java/ai/verta/modeldb/common/authservice/RoleServiceUtils.java

```diff
@@ -85,14 +85,8 @@ public boolean createWorkspacePermissions(
       if (ownerId.isPresent()) {
         setResourcesBuilder.setOwnerId(ownerId.get());
       }
-      if (workspaceId.isPresent()) {
-        setResourcesBuilder.setWorkspaceId(workspaceId.get());
-      } else if (workspaceName.isPresent()) {
-        setResourcesBuilder = setResourcesBuilder.setWorkspaceName(workspaceName.get());
-      } else {
-        throw new IllegalArgumentException(
-            "workspaceId and workspaceName are both empty.  One must be provided.");
-      }
+      workspaceId.ifPresent(setResourcesBuilder::setWorkspaceId);
+      workspaceName.ifPresent(setResourcesBuilder::setWorkspaceName);
```

Comment:
that means that we are changing a logic to make workspaceName more important in request than id? Why do we need it?

Reply:
- In some old endpoint calls, we do not have workspace name as a compulsory field so at that time we do not have workspace Id and Workspace name at MDB. as per @evgeniilezhnin's confirmation, UAC will handle it itself so now I have removed it from compulsory to optional.